framework/status/__init__.py

```diff
@@ -26,10 +26,20 @@ def push_status_message(message, kind='warning', dismissible=True, trust=True, j
     :param dismissible: Whether the status message can be dismissed by the user
     :param trust: Whether the text is safe to insert directly into HTML as given. (useful if the message includes
         custom code, eg links) If false, the message will be automatically escaped as an HTML-safe string.
-    :param jumbotron: Should this be in a jumbstron element rather than an alert
+    :param jumbotron: Should this be in a jumbotron element rather than an alert
     """
     # TODO: Change the default to trust=False once conversion to markupsafe rendering is complete
-    statuses = session.data.get('status')
+    try:
+        statuses = session.data.get('status')
+    except RuntimeError:
```

Comment:
Okay, there's a message I can check. If the message doesn't match, I can re-raise the error. I'll fix that and e if I can add another test case to verify.